src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -10842,16 +10842,16 @@ def dt_freq(self) -> "SnowflakeQueryCompiler":
         return self.dt_property("freq")
 
     def dt_seconds(self) -> "SnowflakeQueryCompiler":
-        return self.dt_property("seconds")
+        return self.timedelta_property("seconds")
```

Comment:
Why change this? why we call timedelta_property for dt_property now? What about the existing code in dt_property?

Reply:
Because initially all properties were calling `dt_property` such that if they were not implemented, a NotImplementedError is raised. This was the case for `seconds` for example before this PR. Now we are distinguishing between datetime properties from timedelta properties, where the former is handled by `timedelta_property` and the latter is handled by `dt_property`.

Reply:
Because initially all properties were calling `dt_property` such that if they were not implemented, a NotImplementedError is raised. This was the case for `seconds` for example before this PR. Now we are distinguishing between datetime properties from timedelta properties, where the former is handled by `timedelta_property` and the latter is handled by `dt_property`.

Reply:
But `dt_property` support `second`, I don't suggest but you could use similar way to support `seconds`. 
I'm a little worry about the code management here. For example `day` go to `dt_property` but `days` go to `timedelta_properties` but esentially they should be the same thing. 
CC @sfc-gh-nkumar Any idea how we can improve this?